rwengine/src/objects/ProjectileObject.hpp

```diff
@@ -26,7 +26,7 @@ class ProjectileObject final : public GameObject {
 
     struct ProjectileInfo {
         ProjectileInfo(ProjectileType p_type, glm::vec3 p_direction,
```

Comment:
Should we keep the `p_` prefixes? It is expected that the constructor takes projectile type, direction, velocity, time and weapon data.